govcd/api_vcd_test.go

```diff
@@ -1979,49 +1978,6 @@ func (vcd *TestVCD) findFirstVapp() VApp {
 	return *vapp
 }
 
-// Test_NewRequestWitNotEncodedParamsWithApiVersion verifies that api version override works
-func (vcd *TestVCD) Test_NewRequestWitNotEncodedParamsWithApiVersion(check *C) {
```

Comment:
Split the test to separate file so that it does not get triggered with `tm` tag. It is an XML testing function, that shouldn't be together with "testing infrastructure functions"